docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  app:
+    build:
+      context: .
+    command: bundle exec rails server --bind 0.0.0.0
+    ports:
+      - 3000:3000
+    volumes:
+      - .:/var/app
+      - ~/Music:/var/music
```

Comment:
It might be out of the scope of this Pull Request, but I believe it's important for the Music directory be configurable from outside of the container.

It seems that the original author's intention was to have a custom initializer (`config/initializers/music_paths.rb`) that could be configured by the user. Since we're using Docker that has to be set up a little differently. I believe that could be achieved by using an environment variable in the `docker-compose.yaml` file.

Reply:
Yes! This can be a new issue to be solved.